### PR TITLE
[LS] Support new import syntax

### LINKED ISSUE #50: [LS ]Support the new import schema
Support the new import schema as part of the flowkit implementation https://github.com/onflow/flow-cli/issues/711
LS should know how to resolve the imports using the existing flowkit implementation.

### PR DESCRIPTION
**Pull request by sideninja**
Closes #50 

## Description
Support the new import schema as implemented in flowkit. It requires adding accounts from configuration to the set of accounts (besides the auto-created ones) and then updating to latest flowkit which supports the new schema resolution.

______

- [x] Targeted PR against `master` branch
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work
- [x] Code follows the [standards mentioned here](https://github.com/onflow/cadence-lint/blob/master/CONTRIBUTING.md#styleguides)
- [x] Updated relevant documentation 
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Added appropriate labels

### CODE REVIEW

**Comment by turbolent** [comment]
I was under the impression that with https://github.com/onflow/flow/blob/master/flips/2022-03-23-contract-imports-syntax.md, the proposed syntax for contracts was using string literals, not identifiers, e.g `import  "Foo"` or `import Foo from "Foo"`. 

So far we are using identifiers solely for standard library contracts, like `Contract`, `RLP`, `BLS`, `Test`, and hopefully more in the future. It would be good to differentiate those from "user contracts" and avoid clashes.

That was also my feedback on an early version of https://www.notion.so/dapperlabs/CLI-development-environment-c5adbf5fba3745138118a64c35f252b0 (though maybe that was on another document, because I can't find my comments there; it was the one we discussed with Peter)

**Comment by sideninja** [comment]
> I was under the impression that with https://github.com/onflow/flow/blob/master/flips/2022-03-23-contract-imports-syntax.md, the proposed syntax for contracts was using string literals, not identifiers, e.g `import "Foo"` or `import Foo from "Foo"`.
> 
> So far we are using identifiers solely for standard library contracts, like `Contract`, `RLP`, `BLS`, `Test`, and hopefully more in the future. It would be good to differentiate those from "user contracts" and avoid clashes.
> 
> That was also my feedback on an early version of https://www.notion.so/dapperlabs/CLI-development-environment-c5adbf5fba3745138118a64c35f252b0 (though maybe that was on another document, because I can't find my comments there; it was the one we discussed with Peter)

Right I remember and also the first implementation was that way but then I learned I couldn't use it along with the "from" I believe, (sorry a bit fuzzy) so the syntax `import "Foo" from "alice"` wasn't working (or at least how I remember) and then I had to switch to `import Foo from "alice"` but then a bit later decided to drop `from "alice"` based on community feedback, and then probably didn't think of changing to `import "Foo"`. We can change this for sure still, the whole purpose of this period is to change things as we find them. I can see why would you like to change it and I agree. Let me try changing it and report back.

**Comment by turbolent** [comment]
@sideninja 
>  Let me try changing it and report back.

Did you have any luck changing from `IdentifierLocation` to `StringLocation`? I can assist if you're running into issues.

File: languageserver/server/server.go [hunk 1]
```diff
@@ -145,6 +145,9 @@ type AddressContractNamesResolver func(address common.Address) ([]string, error)
 // StringImportResolver is a function that is used to resolve string imports
 type StringImportResolver func(location common.StringLocation) (string, error)
 
+// IdentifierImportResolver is a function that is used to resolve identifier imports supported by flowkit
+type IdentifierImportResolver func(location common.IdentifierLocation) (string, error)
+
```
Comment (turbolent) [inline languageserver/server/server.go:150]: Looks like this isn't necessary anymore:

```suggestion
```

File: languageserver/server/server.go [hunk 2]
```diff
@@ -172,6 +175,8 @@ type Server struct {
 	resolveAddressContractNames AddressContractNamesResolver
 	// resolveStringImport is the optional function that is used to resolve string imports
 	resolveStringImport StringImportResolver
+	// resolveIdentifierImport is the optional function that is used to resolve identifiers for the imports used in flowkit
+	resolveIdentifierImport IdentifierImportResolver
```
Comment (turbolent) [inline languageserver/server/server.go:179]: Same here:

```suggestion
```

**Review by turbolent — APPROVED** [review]
Nice!

File: languageserver/server/server.go (hunk 1 above)
Comment (sideninja) [reply languageserver/server/server.go:150]: damn

**Pull request merged**